packages/playwright-test/src/matchers/matchers.ts

```diff
@@ -289,8 +290,13 @@ export async function toBeOK(
 ) {
   const matcherName = 'toBeOK';
   expectTypes(response, ['APIResponse'], matcherName);
-  const log = (this.isNot === response.ok()) ? await response._fetchLog() : [];
-  const message = () => this.utils.matcherHint(matcherName, undefined, '', { isNot: this.isNot }) + callLogText(log);
+  const [log, text] = (this.isNot === response.ok()) ? await Promise.all([
+    response._fetchLog(),
+    response.text()
+  ]) : [];
+  const message = () => this.utils.matcherHint(matcherName, undefined, '', { isNot: this.isNot }) +
+    callLogText(log) +
+    `\nResponse text:\n${colors.dim(text?.substring(0, 1000) || '')}`;
```

Comment:
- Should we only print response if it has `textualMimeType` (we have a helper like that somewhere)? Otherwise, it's probably not even safe to call `response.text()` as it might not be utf8.
- 1000 seems to be a long string, but maybe not 😄

Reply:
> * Should we only print response if it has `textualMimeType` (we have a helper like that somewhere)? Otherwise, it's probably not even safe to call `response.text()` as it might not be utf8.

Agreed, but what if there is no content-type? It's probably good enough to print it if there is no content type or text content-type but that's not ideal either.

> * 1000 seems to be a long string, but maybe not smile

What number do you suggest instead?